_posts/2016/2016-12-20-cqrs-and-elasticsearch.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: post
+title: "CQRS and Elasticsearch"
+date: "2016-12-20 09:00:00"
+image: blog-header/searching.jpg
+categories: tech-stories
+authors: ["Jens"]
+---
+
+After having explained how we use [Multitenancy and Elasticsearch][prev-blog-post], we will continue our series of search-related blog posts by looking at a software architecture pattern called `Command and Query Responsibility Segregation` ([CQRS][fowler-cqrs]) and how we embrace it for managing and accessing product data in our ePages shops.
+
+## Different Product Data Models
+
+In our ePages software we identified two fundamentally different usage scenarios when dealing with product data, each coming with its own data model.
+
+### The Write Model
```

Comment:
The only established term would be "Command and Query Responsibility Segregation", so I leave that in Uppercase while changing the rest to lowercase